src/features/MoviesList/index.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
```

Comment:
something is not used here :-)

Reply:
that's why this all resume-thing is so important  ;) I missed it - already repaired ! :)